sentences/es/light_HassLightSet.yaml

```diff
@@ -4,17 +4,17 @@ intents:
     data:
       # brightness
       - sentences:
-          - "pon [el] brillo de <name> al <brightness>"
-          - "pon <name> (al | con) brillo <brightness>"
-          - "pon <name> a <brightness>"
+          - "(pon|poner|sube|subir|baja|bajar|establece|establecer) [el] brillo de <name> al <brightness>"
```

Comment:
Consider moving this to an expansion rule

Reply:
Done